source/vsub_s8_rvv.c

```diff
@@ -0,0 +1,20 @@
+// Copyright 2024 10xEngineers
+
+#include "../common/rhal_base.h"
```

Comment:
Do not use full relative path, just include the header file only.

Reply:
Fixed